tools/commentMonitor/README.md

```diff
@@ -14,12 +15,15 @@ Running commentMonitor requires the eventmap file which is specified by the `--e
 The `regex_string`, `event_type` and `comment_template` can be specified in the `eventmap.yml` file.
 
 Example content of the `eventmap.yml` file:
-```
+```yaml
 - event_type: prombench_stop
   regex_string: (?mi)^/prombench\s+cancel\s*$
   comment_template: |
     Benchmark cancel is in progress.
 ```
+```shell
+$ export COMMENT_PREFIX='/funcbench,/prombench'
```

Comment:
I would much prefer to have these as cli args.
This way it would also be visible to the generated cli help readme.